src/main/java/edu/kit/datamanager/pit/domain/SimplePidRecord.java

```diff
@@ -0,0 +1,58 @@
+package edu.kit.datamanager.pit.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map.Entry;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SimplePidRecord {
+
+    @JsonIgnore
+    public static final String CONTENT_TYPE = "application/vnd.datamanager.pid.simple+json";
```

Comment:
Note the mimetype for the accept header. It is documented in the swagger UI, but probably need to be adjusted in external examples like in the cookbook.

Reply:
I am considering if it would make sense to switch to media-range parameters instead, but it currently does not have priority. We would need to evaluate the support of spring and openapi/swagger for that. (https://datatracker.ietf.org/doc/html/rfc9110#name-accept)